addons/project/static/src/views/project_task_kanban/project_task_kanban_dynamic_group_list.js

```diff
@@ -34,6 +34,18 @@ export class ProjectTaskKanbanDynamicGroupList extends KanbanDynamicGroupList {
         return result;
     }
 
+    getAdditionalFields() {
+        const model = this.model.root.fields[this.model.root.groupBy].relation;
+        const project_ids = this.context['active_model'] === 'project.project' && this.context['active_ids'];
+        if (model === "project.task.type" && project_ids) {
+            return {
+                'project_ids': project_ids,
+            };
+        }
+
+        return false;
+    }
```

Comment:
```suggestion
    getAdditionalFields() {
        const res = super.getAdditionalFields();
        const model = this.model.root.fields[this.model.root.groupBy].relation;
        const project_ids = this.context['active_model'] === 'project.project' && this.context['active_ids'];
        if (model === "project.task.type" && project_ids) {
            res.project_ids = project_ids;
        }
        return res;
    }
```
Or am I missing something?